packages/neo-one-node-blockchain/src/Blockchain.ts

```diff
@@ -390,7 +391,8 @@ export class Blockchain {
     }
   }
 
-  public async invokeScript(script: Buffer): Promise<CallReceipt> {
+  public async invokeScript(script: Buffer, signers?: readonly Signer[]): Promise<CallReceipt> {
+    // TODO: implement with new signers
     const transaction = new Transaction({
```

Comment:
not sure what this change is but you can probably remove any changes you made to node-blockchain